internal/catalogmetadata/types.go

```diff
@@ -129,6 +129,9 @@ func (b *Bundle) loadMediaType() error {
 }
 
 func (b *Bundle) propertyByType(propType string) *property.Property {
+	// TODO: Get rid of the map and support multiple properties with the same type
+	// Some property types such as `olm.gvk` and `olm.gvk.required`
+	// can appear in the b.Properties list multiple
```

Comment:
Is there an issue for this?

Reply:
I created a PR for this: https://github.com/operator-framework/operator-controller/pull/427

Will remove the to do from this PR soon as it affects nothing here.